uint8 tag_id          # 태그 번호 (1~10)
uint16 position_x     # x 좌표 (0~1900)
uint16 position_y     # y 좌표 (0~1900)
uint8 vehicle_type    # 2: 장애인차, 1: 전기차, 0: 일반차
